src/ecl_data_io/read.py

```diff
@@ -38,6 +38,17 @@ def lazy_read(filelike, fileformat=None):
     :param fileformat: Either ecl_data_io.Format.FORMATTED for ascii
         format, ecl_data_io.Format.UNFORMATTED for binary formatted files
         or None for guess.
+
+    :raises ecl_data_io.EclParsingError: If the file is not a valid
+        ecl file.
```

Comment:
Is `ecl` always referring to Eclipse? If so, why don't we write the full name?

Reply:
ecl is the name of the group that made eclipse. Its a convention now, perhaps something to change, but that is perhaps a bigger issue than this PR as ecl is in the name of the package.